examples/lvgldemo/Makefile

```diff
@@ -86,11 +86,18 @@ endif
 
 MAINSRC = lvgldemo.c
 
-LVGL_SRC_DIR=$(APPDIR)/graphics/lvgl/lvgl
-LVGL_DIR=$(APPDIR)/graphics/lvgl
+# Upstream `lv_examples/lv_examples.h` includes LVGL
+# in following way `#include "../lvgl/lvgl.h"`, for no clean reason.
+#
+# The way that we export LVGL offers `#include <lvgl/lvgl.h>`, so it won't work.
+# Solution below is not pretty but it fixes `lv_examples` without patching.
+#
+# Remove this fix when you will get better idea.
+
+LVGL_EXAMPLE_INCLUDE_FIX=$(APPDIR)/graphics/lvgl/lvgl
```

Comment:
We can mdoify the inclusion to lvgl.h by LV_LVGL_H_INCLUDE_SIMPLE macro like this:
CFLAGS += ${shell $(DEFINE) "$(CC)" LV_LVGL_H_INCLUDE_SIMPLE}
Here is the related code: https://github.com/lvgl/lv_examples/blob/master/lv_examples.h#L16

Reply:
For this to work I added one more export in LVGL. Now LVGL can be included both with `lvgl/lvgl.h` and `lvgl.h` paths.

`CXXFLAGS` as well, right?

Done.